Library/Homebrew/test/cmd/info_spec.rb

```diff
@@ -26,6 +39,13 @@
       .and be_a_success
   end
 
+  it "check --json=v2 format", :integration_test do
+    setup_test_formula "testball"
+
+    expect { brew "info", "testball", "--json=v2" }
+      .to output(expected_output).to_stdout
+  end
+
```

Comment:
Let's not add this as integration tests are super slow. Instead, how about using the tests in [`formula_spec.rb`](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/test/formula_spec.rb):

https://github.com/Homebrew/brew/blob/f416f8b54cec576a6528447f16b9b9bdd4df4b7d/Library/Homebrew/test/formula_spec.rb#L826-L843

Reply:
Done! I have removed the original test.